### DISCUSSION TITLE
Feature Request regarding system timeout for server plugin scan

### DISCUSSION THREAD

**Discussion by LaddieErvin** [apohl79/audiogridder#415] [Ideas]
If possible, could you please add a feature to allow users to disable the automatic timeout during the server plugin scan? I have enough plugins to scan that that window pops up and halts the process no fewer than six times. This means, whenever I run  plugin scan, I have to babysit the process and press "Wait" at least six times. This is time consuming and I don't understand how this timeout feature benefits the user. I can only guess that it's there to allow users to "Abort" a scan, if they wish, but I'd personally rather have to use CTRL+ALT+DLETE to stop a scan (if ever necessary) than have to sit and wait for the pop-up six times (or more) when running a scan - which I need to do just about every week to stay current.

Thanks for your time and consideration and for creating this wonderful tool.

**Comment by apohl79** [comment]
Understood. I think there should be an option in the settings than and also the possibility to disable the dialogue at the first appearance, as you will probably want to change this on a fresh install too.

I will create a feature request ticket.

> I don't understand how this timeout feature benefits the user.

The main intention is to allow a user to run through activation steps for new plugins, which can take long. The timeout itself was intended for your exact use case: to keep it finish without user interaction. I guess the question screen kind of broke this behavior. ;-)

**Reply by AtmanActive** [reply]
Then we need timeout after timeout.
So, yes, the timeout happens and the dialog appears, but, then, that dialog counts down, for example, 60 seconds, and self-closes to continue with the scanning.
